stories/__tests__/SQFormDialog.stories.test.js

```diff
@@ -168,3 +168,40 @@ describe('Tests for WithAutoFocus', () => {
     expect(textfield).toBe(document.activeElement);
   });
 });
+
+describe('Tests for Tertiary Button', () => {
+  it('should find the tertiary button', async () => {
```

Comment:
We always want to speak in terms of what the user would see or experience, rather than the test itself.

'Should display the tertiary button'